### DISCUSSION TITLE
《用户行为分析：如何用数据驱动增长》读书分享

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#7] [4️⃣ 新媒体]
**基本信息：**

书名：《用户行为分析：如何用数据驱动增长》
作者：张溪梦、邢昊 等
出版时间：2021 年
和我们的关系：开发者行为数据分分析与场景化，可以很好的借鉴电商场景，包括分析框架、方法论、案例等

评论信息：

京东评论：https://item.jd.com/13229632.html
豆瓣评论：https://book.douban.com/subject/35506029/

<img src="https://user-images.githubusercontent.com/15010826/158515427-ce1edf15-019c-4956-9e3b-f1ce898ee9d0.png" width="200px">

**Comment by zhuxiangning** [comment]
先同步下第一次分享的内容～欢迎大家多多互动～
一、什么是用户行为数据
1、狭义和广义的用户行为数据

**狭义的用户数据**
用户属性数据：指用于描绘用户特征的数据，根据人口统计学特征，可以从静态、动态、未来的发展趋势三个方面观察（会员信息管理系统）
用户行为数据：指用户在商业互动过程中产生的动作数据，即用户做了什么事情，在线下门店或线上商城（门店经营管理系统）
用户交易数据：指用户完成支付之后产生的相关数据，如订单金额，数量，类型等（订单交付管理系统）
![image](https://user-images.githubusercontent.com/59213474/158797486-30b110f1-8861-4e66-8051-e6d2501ce441.png)

**广义的用户数据**
![image](https://user-images.githubusercontent.com/59213474/158797454-396da89e-d963-4d1d-a51a-8e4fc2ad46c1.png)

- 当将交易与用户行为关联之后，在以用户为中心的逻辑下，我们看到的将是用户的整个生命周期：用户的第一笔订单，第一次复购，将成千上万的用户周期总价值叠加在一起，就是企业总的订单和。

- 用户的属性数据可以理解为用户过去行为的沉淀，根据此我们可以将用户属性与用户行为关联，用户浏览商品沉淀了品类偏好，访问门店而沉淀的地理信息，支付订单的平均金额沉淀的消费力属性。

通过以上方式获得的用户属性数据，能够描绘出更加立体，鲜活，有价值的全方位用户画像，为“千人千面”的精细化运营奠定了基础。

2、用户行为数据的“5+1”要素
who\when\where\what\how\how much
谁，在什么时间，什么地点，对什么内容，做了什么事情，产生了多少结果

- who\谁：不仅指用户的姓名、联系方式、身份证号，还包括用户访问终端的信息（手机端、PC端等），ID信息和用户彼此是多对多的关系

- when\何时：行为的时刻和发生的时长

-  where\在哪里：线下坐标信息，线上的用户来源渠道

-  what做什么：可以从内容、产品、功能维度进行细分

- how\怎么做：产品互动（加购、下单、支付等）、内容互动（浏览、滑动、点赞、评论、分享等）和功能互动

-  how much\做多少：把订单金额、订单类型、商品明细等信息上传到用户行为分析的体系里，进行用户行为数据的融合探索

3、用户行为数据的一个趋势和两个价值

- 趋势：在数字孪生（digital twin）世界下，用户行为数据的迁移

需求侧的数字孪生即数据化营销是当下我国在数字化转型方面发展最快的地方。供给侧的数字化能降低企业的bottom line（成本线），需求侧的数字化能提升企业的top line（营收线）
不可改变的趋势：物理世界正在不可改变地向数字世界迁移，用户的行为也必然随之迁移。谁先能够率先采集、识别、分析这些行为数据，谁就掌握了打开数字孪生世界经营的🔑。
![image](https://user-images.githubusercontent.com/59213474/158797377-8af75344-9117-4432-bd78-2928700e65b1.png)

-  价值：
- 理解用户需求，指导业务升级：如何用 用户行为数据 理解用户需求
![image](https://user-images.githubusercontent.com/59213474/158797312-35843b9c-6557-43f4-b28c-aafd8c45fdd9.png)

描述用户：落在不同的生命阶段的用户会呈现出不一样的用户价值
![image](https://user-images.githubusercontent.com/59213474/158797280-527f353f-7071-4d8d-898a-1d4534722b02.png)

理解需求：通过A/B test，观察对照组和实验组的数据差，判断用户的真实需求

设计业务：强调用户旅程地图。这是将一个人在与企业接触的过程中，为了完成某个目标所经历的旅程可视化的一种工具。
用户旅程地图本身便是由一系列用户行为在时间线上的排布构成的。用户旅程地图的格式，上部包括用户角色、业务场景；中部是不同时期用户行为、想法、情感及相对应的期望和目标；底部是收获，包括洞察及对应的服务资源。前几年，当我们引入用户旅程地图描述用户行为的时候，还没有太多的数据基础，更多的是一种主观的 UX（用户体验）设计方法。如今，借助用户流转地因，已经可以客观地将用户行为在已有业务流程上连贯地表示出来，方便业务管理者发现业务的瓶颈、找到新的通路。

重建关系：传统企业：人——货——场；互联网企业：AARRR（Acquisition：获取用户，Activation：提高活跃度，Retention：提高留存率，Revenue：获取收入，Refer：自传播）

-  预测用户行为，引导业务创新

用户行为数据的应用，从原始数据处理，到标准报告生成，到专题报告，再到BI（商业智能）分析报告，这几个阶段基本体现了数据的第一个价值，它可以告诉企业管理者自己的业务正在发生什么、是什么导致了结果的产生，属于业务提升的必要手段。
![image](https://user-images.githubusercontent.com/59213474/158797233-b59430bb-5e3d-42ab-8c9a-001ef2649420.png)

企业业务创新发展的过程可以分为**三个关键环节**：
![image](https://user-images.githubusercontent.com/59213474/158797184-6b897139-6949-4b83-99dd-a2930cd5e046.png)

- 预测未来：精准预测->信息茧房

- 精准营销：相似人群扩展（look-alike）算法

“我知道广告费有一半浪费了，但是我不知道是哪一半”
“我知道你把转化率提高了5倍，但你投放的人群缩小了10倍”
精准营销的隐藏属性——高频
如果能够实现“高频”，那么质疑精准营销的问题其实就有了答案：“我确实把投放量缩小了 10倍，但这样的派缩人群包，按照不同的维度，我准备了1000个，已经将全量人群覆盖了 10 遍，而且随着用户行为数据的反馈，我还可以每天选代生成1000个新的人群包，并用算法择优推送营销方案。”这样的完美答案听起来就很霸气，但对于企业拥有的数据量、模型能力、迭代速度，甚至个性物料创意的能力都是巨大的挑战。

- 评估用户价值：CLV （customer lifetime value, 客户生命周期价值）

海外电商平台巨头亚马逊，就是使用这种方式来评估用户价值的，它们根据用户的行为数据建模，计算出每个用户一生中会在亚马逊消费多少钱，即CLV。基于这样的价值，它们会关注每次促销活动对单个用户CLV的影响，而不仅仅是促销对转化订单的影响。这种全新的评估方式，使得企业会兼顾转化率与,忠诚度的变化，平衡短期效益和中长期收益的冲突。
基于用户行为数据评估用户生命周期总价值，企业的关注点在空间维度可以从单一的交易环节放大至用户在平台互动的全流程，在时间维度可以从用户初次接触，延续到永远流失的完整生命周期。这是一种全新的企业经营视角，也是一种商业文化进步的体现。

4、用户行为数据驱动增长的困境

- 提不出需求

- 采集工作缺失（数字化&数据化）

- 有报表无洞察

- 有洞察无策略

5、数据驱动增长的金字塔
规划——采集——分析——应用
![image](https://user-images.githubusercontent.com/59213474/158797123-60a43ea1-160a-489f-989f-a23d25612049.png)

**Comment by zhuxiangning** [comment]
更新第二波～

二、如何进行数据规划

-  OSM（Objective-Strategy-Measure）
- +UJM（User Journey Map）
- +场景化

以电商产品为例

**数据规划：基于用户行为数据，规划指标体系**

1、数据规划的常见问题
1. 不知道看什么数据
2. 想看数据，却没有数据
3. 缺少导向结果的数据依据->过多关注结果性数据，忽略了过程性数据，会导致数据归因的困难
4. 数据没有治理->没有统一数据口径，没有建立规范的数据标准
5. 数据目标未对齐->从公司战略目标出发，拆解到各个职能部门，大家为各自部门的数据目标负责的同时，也应该与公司战略目标对齐

2、如何进行数据规划

**OSM模型**
Objective, Strategy, Measure。O（objective）代表目标，需要我们思考或者回答，我们的业务、产品，甚至是产品中的一个小功能存在的目的是什么？能够解决用户的什么问题？满足用户的什么需求？如果你是公司负责人，则需要思考公司的核心目标是什么，可能是公司今年的营收、利润额等；如果你是产品部门负责人，则需要思考未来几年的产品发展方向、产品规划、产品路线图等。

S（Strategy）代表策略，是指清楚目标之后，为了达成目标，我们应当采取的各种策略。

M（Measure）代表度量，用于衡量我们的策略是否有效，反映目标的达成情况。度量涉及两个概念，
一个是KPI（关键绩效指标），用来直接衡量策略的有效性；一个是目标值（Target )，需要预先给出数值，来判断策略是否能达到预期值。

这里以某 App 的非标准住宿搜索场景为例，介绍如何从负责搜索功能的产品经理的角度出发，将OSM 模型应用于实践，规划指标体系。
非标准住宿有一个特点，就是产品个性化。比如，用户在住宿的时候会选择情侣房、家庭房等个性化佳房，但其数量是有限的，同一种风格的房源可能不会超过5间。

一般情况下，非标准住宿搜索预定的完整路径为：发起搜索一›搜素结果列表一产品详情页一>填写订单一>预订成功。

在这个业务流程中，我们该如何用 OSM 模型规划指标体系呢？

第一步：选定目标 ( Obiective)
在该场景中，选定目标可以应用两种视角：第一种，用户视角，让用户通过搜索功能高效找到自己心仪的房间；第二种，业务视角，通过提高搜索成功率，进而提升用户的下单转化率。
对于如何更好地选定目标，可以首先将公司核心KPI 目标拆解到业务线的核心目标，其次还要关注长期目标和近期目标，并让其相辅相成。在选定目标时，需要注意四个原则，即选定目标的DUMB 原则：切实可行
(Doable）、易于理解（Understandable）、可干预可管理（Manageable )、正向有益的(Beneficial )。我们可以发现，上述从用户视角和业务视角出发选定的两个目标，都是符合 DUMB 原则的。

第二步：采取策略 ( Strategy )
那么，为了达成以上两个目标，根据经验判断，我们可以采取相应的策略。策略一，提高返回结果与用户搜索词的匹配度；策略二，提供有效的搜索结果排序，对于非标类产品，我们需要思考怎样把用户感兴趣的产品放在第一屏或者前三位，以便让用户一眼就看到想搜索的产品；策略三，当搜索没有结果或者结果显示数量不足时，我们要做有效的推荐。

第三步：有效度量 ( Measure)
用 KPI 可以直接衡量策略的有效性。针对第二步 “采取策略”中提出的三个策略，根据从搜索到下单的整个业务流程，我们可以拆解出两个 KPI 指标。
KPI 1：搜索到详情页的转化率，我们设置的目标值 （Target）是 X%，在这一步，用户通过搜索得到想要的结果，才会点击产品的详情页。
KPI 2：从详情页到下单的转化率，我们设置的目标值 （ Target）也是X%，在这一步，如果用户看到的产品详情页是符合心意的，才可能会产生订购，达到最后的预订成功。

KPI1 和KPI2 都是过程性指标。过程性指标，可以简单理解为我们到达这个结果之前经过的路径，以及通过这个路径去衡量转化结果好坏的过程，它是可干预的，而且通常是用户行为。此外，还需要注意结果性指标。

结果性指标，最常见的就是电商场景中的 GMV（成交总额）或订单量，它通常出现在业务漏斗的底部，是一个不可更改的、后验性的指标。

通过优化过程性指标可以改善结果性指标的表现。在非标准住宿用户从搜索到下单这个场最中，结果性指标是 “订单金额”，或者 “提交订单数量”

**UJM模型**
UJM（User Jourey Map）是用户旅程地图的意思。UIM 模型是我们在设计一款产品的过程中，必须要去梳理的。

我们通过OSM 模型确定目标（O）、策略（S）和度量（M）后，需要回过头来梳理整个产品的用户旅程地图，因为在UJM 中的不同阶段都对应不同的目标。比如在用户初次进人产品阶段时，我们的首要目标就是用广留存，为了达到这一目标，就需要寻找用户与产品产生的每一个接触点。每一个接触点的背后都存在痛点和机会点，用来反哺 OSM 模型。

下面以一个简化版电商产品的 UJM 模型为例（见图2-6）介绍。梳理电商产品 UJM 模型的过程包括：拆解用户所处的每一个旅程的阶段、了解每个阶段中的用户行为、明确每个阶段中的电商产品目标、发现各阶段中电商产品与用户的接触点、从接触点里找到痛点和机会点。
用户在使用一款电商平合时，大致会经历六个阶段：从各个途径了解并进人该电商平台，通过首页或搜索功能或商品类目页等其他入口“逛”平台，对商品产生兴趣进人商品详情页，进入付费流程完成一次重要的付费转化，用户主动或者被动分享，产生复购。
![image](https://user-images.githubusercontent.com/59213474/161931456-43dce5f7-cfa2-4e79-bdfd-7249c4c14ddb.png)


**场景化**
看了指标体系的大框架之后，就能直找开始数据采集、数据分析吗？我们在与客户一起实践探素的过程中发现，只有OSM 機型和UJM 模型，在落地 “数据驱动增长”的过程中还有一些掣肘。因为这个框架过于庞大，当我们想要快速切人、快速落地时，往往找不到明确的切人点。

为了将这些高高在上、抽象的战略目标下沉到一线工作人员的具体执行任务当中，我们引进了“场景化”概念。场景化其实是为了帮助我们在庞大的 OSM 模型和 UJM 模型之下，能够模块化、结构化地快速切人和落地指标体系而提出的。也就是说，场景化在这里的作用是推动指标体系的快速落地。

我们在与上千家客户一起落地指标体系的过程中，按照企业内不同部门所对应的用户生命关键旅程，结合当下最具潜力的增长点，梳理出来了能够满足指标体系快速落地的 20 个通用型场景（见图 2-8）。

通过OSM 模型和 UJM 模型规划出指标体系的大框架之后，就可以将这20 个通用型场景分阶段分门别类地对应到用户旅程地图中，满足不同部门、不同层级人员从拉新到转化，最终到提升客单价等环节的需求。

同样以简化版的电商产品为例，在OSM模型与 UJM 模型相结合的基础上，把20个通用型场景中适用于电商产品的场景带人指标体系当中（见图2-9)。这样在用户旅程地图的每个阶段又对号人座了各自关键的场景。
![image](https://user-images.githubusercontent.com/59213474/161931266-98f083bd-57bc-4e7e-8df6-a1cbf7a3f7f0.png)


有了OSM模型和 UJM 模型并匹配到各个细分场景后，整个指标体系就可以从一个非常大的战略目标层层拆解到一线工作人员可落地执行的场景。
